ckanext/xloader/config_declaration.yaml

```diff
@@ -46,6 +46,13 @@ groups:
         type: bool
         required: false
         legacy_key: ckanext.xloader.just_load_with_messytables
+      - key: ckanext.xloader.strict_type_guessing
+        default: True
+        example: False
+        description: |
+            Use with ckanext.xloader.use_type_guessing to set strict true or false
```

Comment:
Can this describe the meaning of strictness in this context?

Reply:
Copying the description from `utils.py` would work; "Strict means that a type will not be guessed if parsing fails for a single cell in the column."

Reply:
Added utils.py method comment into config declaration description.